### ISSUE TITLE
New feature：highlight a node's neighborhood when it is clicked

### ISSUE THREAD

**Issue by liutf2005** [Yomguithereal/ipysigma#2]
Hi  Yomguithereal,

As I mentioned before,  I think it will be cool to add the following feature in ipysigma:

"... highlight a node's neighborhood when it is clicked, by drawing every other nodes as grey."

This feature is also introduced in the webpage of sigma.js :  http://sigmajs.org/,  you can take a look.

Thanks,

**Comment by liutf2005** [comment]
Hi Yomguithereal,

Any progress on this issue？

btw. I also introduce ipysigma to my colleagues and they think it is cool too.

**Comment by Yomguithereal** [comment]
Just published a v0.5.0 with the feature. Tell me what you think.

**Comment by liutf2005** [comment]
Great. Thanks for your work!

I tried the new version today and found that : 

(1)  automatic layout function is gone：in the previous version，there is a play button at the corner to activate the layout function. I cannot find it now.  I think the layout algorithm should be one of the core components, it can provide better visualization.  

(2) the highlight function: when I click one node，only the selected node and its neighbors appear in the graph and other nodes do not appear. How about keep all the nodes in the graph and differentiate them in different colors? For example, the selected node and its neighbors can be red, and the rest of the nodes can be light grey. In this way, we can see how the selected ones are related to the rest of nodes in the big picture.  

(3) label of the nodes：in the previous version and the latest version, some labels of the nodes will appear, but not all of them. And it seems that it is random.  How about hiding all labels by default and showing the label only when mouse over it ？

Recently I also tried other visualization tools for network analysis.  I still think your product is my best choice : )  Again, thanks for your work.
